code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -95,6 +95,10 @@
 			if(!get_location_accessible(H, "mouth"))
 				user << "<span class='warning'>The mask is in the way.</span>"
 				return
+			if(H.species && H.species.flags & ALL_RPARTS) //If the target is of a species that can have prosthetic heads, but doesn't have one...
+				if(!(H.rparts_data["head"]))

```

Comment:
not expressions should not have brackets unless it is a multi-part expression or it is using the `in` operator